### PR TITLE
Feature/ccit 128 ess interaction on contact page

### PR DESCRIPTION
**Pull request by richmadetech**
## Description of change

ESS Interactions show on the Company activity Feed already. This PR also shows ESS Interactions on the CONTACT activity feed. 

## Test instructions

Go to a company with ESS interactions, e.g. One List Corp, select a contact from the ESS Interaction (e.g. Test Test), view their Interactions on their Contact page. The same ESS interaction will appear in the CONTACT activity feed. 
e.g. http://localhost:3000/contacts/c1376844-89da-409c-8194-13e0b509b9d0/interactions

### Before
Screenshot 2023-01-30 at 15 17 54

### After
Screenshot 2023-01-30 at 15 16 41

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'

- [x] Has the branch been rebased to main?
- [x] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, Edge, Safari)

### CODE REVIEW

File: src/apps/companies/apps/activity-feed/controllers.js [hunk 1]
```diff
@@ -355,24 +362,15 @@ async function fetchActivityFeedHandler(req, res, next) {
...
-          ]
-          activity.object.startTime = activity.published
-        }
+      if (isEssActivity(activity)) {
+        const essContactEmail = activity.actor['dit:emailAddress']
+        const essContact = getContactFromEmailAddress(
+          essContactEmail,
+          company.contacts
+        )
+        activity = augmentEssActivity(activity, essContact)
```
Comment (richmadetech) [inline src/apps/companies/apps/activity-feed/controllers.js:371]: This function is checked independently and in the functional tests

File: src/apps/companies/apps/activity-feed/controllers.js [hunk 2]
```diff
@@ -355,24 +362,15 @@ async function fetchActivityFeedHandler(req, res, next) {
...
-
-          activity.object.attributedTo = [
-            activity.object.attributedTo,
-            mapEssContacts(essContact),
-          ]
-          activity.object.startTime = activity.published
-        }
+      if (isEssActivity(activity)) {
+        const essContactEmail = activity.actor['dit:emailAddress']
+        const essContact = getContactFromEmailAddress(
```
Comment (richmadetech) [inline src/apps/companies/apps/activity-feed/controllers.js:367]: covered by functional tests

File: test/sandbox/fixtures/v4/activity-feed/data-hub-and-external-activities.json [hunk 3]
```diff
@@ -409,20 +373,18 @@
         "_source": {
           "actor": {
             "dit:blackListedReason": null,
-            "dit:emailAddress": "contact@bob.com",
+            "dit:emailAddress": "superglue@gov.uk",
             "dit:isBlacklisted": false,
```
Comment (richmadetech) [inline test/sandbox/fixtures/v4/activity-feed/data-hub-and-external-activities.json:377]: main Change - The rest was Prettier Formatting

**Review by PaulWheatcroft — APPROVED** [review]

**Review by onyela-o — APPROVED** [review]
LGTM :)

**Pull request merged**